tests/samples/example.xml

```diff
@@ -0,0 +1,237 @@
+<article
```

Comment:
@Rossi-Luciano evite adicionar o XML inteiro. Só adicione no próprio teste o que será testado. 
Se for necessário inserir o XML inteiro, coloque em fixtures como padrão.

Reply:
OK, @robertatakenaka